epochStart/shardchain/trigger.go

```diff
@@ -276,6 +289,36 @@ func (t *trigger) ReceivedHeader(header data.HeaderHandler) {
 		return
 	}
 
+	t.tryUpdateTriggerFromMeta(metaHdr, hdrHash)
+}
+
+func (t *trigger) tryUpdateTriggerFromMeta(metaHdr *block.MetaBlock, hdrHash []byte) {
+	if !metaHdr.IsStartOfEpochBlock() {
+		t.updateTriggerFromMeta(metaHdr, hdrHash)
+		return
+	}
+
+	for _, miniblock := range metaHdr.MiniBlockHeaders {
```

Comment:
this check of having miniblocks or not is not needed here.

Reply:
removed